### PR TITLE
feedbackendpoint

### PR DESCRIPTION
**Pull request by ZEEZKING**

### CODE REVIEW

File: Application/Exceptions/NotFoundException.cs
Comment (ZafrullahId) [inline Application/Exceptions/NotFoundException.cs:1]: NotFoundException ==> FeedBackNotFoundException

File: Application/Features/FeedBack/CreateFeedBack/CreateFeedbackCommandHandler.cs
Comment (ZafrullahId) [inline Application/Features/FeedBack/CreateFeedBack/CreateFeedbackCommandHandler.cs:1]: You are voilating DDD principle.
The create method should be in the Feedback entity class. 
Check the User class for example

Comment (ZafrullahId) [reply Application/Features/FeedBack/CreateFeedBack/CreateFeedbackCommandHandler.cs:1]: You are not allowed to create a constructor of the Feedback entity. The constructor should be private

File: CBT_PrebCenter/Endpoints/FeedBack/CreateFeedback/CreateFeedbackResponse.cs
Comment (ZafrullahId) [inline CBT_PrebCenter/Endpoints/FeedBack/CreateFeedback/CreateFeedbackResponse.cs:1]: What about BaseApiResponse

File: Application/Features/FeedBack/CreateFeedBack/CreateFeedbackCommandHandler.cs
Comment (ZEEZKING) [reply Application/Features/FeedBack/CreateFeedBack/CreateFeedbackCommandHandler.cs:1]: > You are voilating DDD principle. The create method should be in the Feedback entity class. Check the User class for example

didiworkontheentityclassareunottheworkthatworkedinitandudidntaddituadditinstudentclass

Comment (ZEEZKING) [reply Application/Features/FeedBack/CreateFeedBack/CreateFeedbackCommandHandler.cs:1]: > You are not allowed to create a constructor of the Feedback entity. The constructor should be private

ididntcreateanyconstructor

**Pull request merged**